backend/baseclass.pm

```diff
@@ -35,6 +35,7 @@ use OpenQA::NamedIOSelect;
 
 use constant FULL_SCREEN_SEARCH_FREQUENCY => $ENV{OS_AUTOINST_FULL_SCREEN_SEARCH_FREQUENCY} // 5;
 use constant FULL_UPDATE_REQUEST_FREQUENCY => $ENV{OS_AUTOINST_FULL_UPDATE_REQUEST_FREQUENCY} // 5;
+use constant DEFAULT_FFMPEG_CMD => 'ffmpeg -hide_banner -nostats -r 24 -f image2pipe -vcodec ppm -i - -pix_fmt yuv420p';
```

Comment:
I suggest to replace that constant with a function like

```perl
sub ffmpeg_cmd() { $bmwqemu::vars{FFMPEG_CMD} // 'ffmpeg -hide_banner -nostats -r 24 -f image2pipe -vcodec ppm -i - -pix_fmt yuv420p' }
```

Reply:
I don't think this is a useful customization point. We already have one via the worker config and adding and additional one that is very redundant is causing more confusion than it helps.

Reply:
You mean one can customize the external video encoder command line?

Reply:
Yes, via `EXTERNAL_VIDEO_ENCODER_CMD`.

Reply:
right. So then it's not strictly necessary to make the rest configurable